core/services/ocr2/plugins/ocr2keeper/integration_test.go

```diff
@@ -738,8 +739,8 @@ func TestFilterNamesFromSpec21(t *testing.T) {
 	require.NoError(t, err)
 
 	assert.Len(t, names, 2)
-	assert.Equal(t, logpoller.FilterName("OCR2KeeperRegistry - LogProvider", address), names[0])
-	assert.Equal(t, logpoller.FilterName("EvmRegistry - Upkeep events for", address), names[1])
+	assert.Equal(t, logpoller.FilterName("KeepersRegistry LogProvider", address), names[0])
+	assert.Equal(t, logpoller.FilterName("KeeperRegistry Events", address), names[1])
```

Comment:
nit. this is probably worthy of its own test file, i was confused why 2.0 integration test is affected by this PR